### PR TITLE
Fixed edge case when vehicle array is empty

### LINKED ISSUE #1126: Public test error - A3A_fnc_getVehiclePoolForAttacks
This error pops up multiple times in the rpt.
Current log is uploaded in the correct discord channel.

```
21:07:17 Error in expression <) then
{
private _points = (_x select 1)/(count (_x select 0));
{
if(([_x, _filt>
21:07:17   Error position: </(count (_x select 0));
{
if(([_x, _filt>
21:07:17   Error Zero divisor
21:07:17 File functions\Base\fn_getVehiclePoolForAttacks.sqf [A3A_fnc_getVehiclePoolForAttacks]..., line 332
```

### PR DESCRIPTION
**Pull request by wurzel0701**
If thats not the case, something strange is happening and more information are needed to solve that

## What type of PR is this.
1. [X] Bug
2. [ ] Enhancement

### What have you changed and why?
Information:
Fixed an edge case when a vehicle array is defined but empty for a modset, resulting in the vehicle pool selection code crashing. 

I added a check if the amount of vehicles is equal zero and if so, the code will log it. Added further logs to find out the array in question.

If that is not the reason for the issue, the new logs should improve debugging abilities.

### Please specify which Issue this PR Resolves.
closes #1126 (hopefully)

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the Mission in Singleplayer?
2. [ ] Have you loaded the Mission in a Dedicated Server?

### Is further testing or are further changes required?
1. [X] No
2. [ ] Yes (Please provide further detail below.)

********************************************************
Notes:

### CODE REVIEW

File: A3-Antistasi/functions/Base/fn_getVehiclePoolForAttacks.sqf [hunk 1]
```diff
@@ -329,7 +332,16 @@ private _vehiclePool = [];
...
+        private _points = 0;
+        private _vehicleCount = count (_x select 0);
+        if(_vehicleCount != 0) then
+        {
+            _points = (_x select 1)/_vehicleCount;
+        }
+        else
+        {
+            [1, "Found vehicle array with no defined vehicles!", _fileName] call A3A_fnc_log;
+        };
```
Comment (jaj22) [inline A3-Antistasi/functions/Base/fn_getVehiclePoolForAttacks.sqf:344]: As discussed, these errors shouldn't be thrown because it's working as intended. AFRF and TKM have empty arrays for vehCSATTransportPlanes because those modsets lack anything suitable. Any other WW2-era mod that we eventually support will have multiple empty arrays. It's just a case that needs to be handled.
